fxmanifest.lua

```diff
@@ -11,7 +11,14 @@ server_scripts {
     'server/main.lua'
 }
 
-client_script 'client/main.lua'
+client_script {
+	'@PolyZone/client.lua',
```

Comment:
```suggestion
    '@PolyZone/client.lua',
```
Incorrect linting / spacing